openslides/assignments/static/templates/assignments/assignment-detail.html

```diff
@@ -133,7 +133,13 @@ <h3 translate>Election result</h3>
         index="$index" heading="{{ 'Ballot' | translate }} {{ assignment.polls.length - $index }}">
       <div ng-controller="AssignmentPollDetailCtrl">
         <!-- action buttons -->
-        <div class="pull-right">
+        <div class="pull-right" os-perms="assignments.can_manage">
+          <span class="spacer-right" editable-text="poll.description" onaftersave="saveDescriptionChange(poll)">
+            <translate>Comment on the ballot paper</translate>:
```

Comment:
Suggestion: Set this label as tooltip. Otherwise it's very long line with label and inline editing value...
Use 'hint' instead of 'comment' (it's not an internal comment).